equipment_handler/source/skytraq_gps_driver.c

```diff
@@ -93,6 +93,21 @@ GPS_RETURNSTATE gps_configure_message_types(uint8_t GGA, uint8_t GSA, uint8_t GS
  */
 GPS_RETURNSTATE gps_disable_NMEA_output() { return gps_configure_message_types(0, 0, 0, 0); }
 
+/**
+ * @brief Get CRC of software on the skytraq
+ * @param crc pointer to uint16_t to store crc
+ *
+ * @return GPS_RETURNSTATE
+ */
+GPS_RETURNSTATE gps_skytraq_get_software_crc(uint16_t *crc) {
```

Comment:
Is this really necessary? Why not just have skytraq_query_software_CRC accept a crc parameter and do the reply parsing in there?

Reply:
This just follows what the code already did with other things. As to why I made it that way originally, I didn't know much C when I wrote this